### DISCUSSION TITLE
Migrate Terraform/Terragrunt managers to off-the-shelf parser

### DISCUSSION THREAD

**Discussion by secustor** [renovatebot/renovate#13465] [Suggest an Idea]
I have been looking at this parser: 
 -    https://www.npmjs.com/package/@cdktf/hcl2json

This would simplify our parsing of HCL/TF files, but `hcl2json` requires an external binary https://github.com/tmccombs/hcl2json

Context: https://github.com/renovatebot/renovate/issues/13461

WDYT: @rarkins @viceice

**Comment by rarkins** [comment]
I definitely don't like binaries, especially in a manager where I hope one day to allow a lot of functions be run in the browser too.

@zharinov you've done a lot of parsing work lately, wdyt?
